src/reference/interactive/index.pug

```diff
@@ -86,62 +88,67 @@ block reference
+    p A Control is a clickable, interactive image within a scene. #[a(href='#participants') Participants] can interact with the control using keyboard, mouse, or controller. There are currently has two types of controls - Buttons and Joysticks. Additional control types may be added in the future.
 
     h4#buttons Buttons
-    .figure
-        img(src='/img/reference/interactive/button.png' alt='A button control as displayed to a participant. This button costs one spark to press.')
-    p.caption A button control as displayed to a participant. This button costs one spark to press.
-    p Buttons are a rectangle positioned within a scene that a participant can click with their mouse or trigger with their keyboard or controller. #[a(href='#the-gameclient') Game Clients] receive events when a button is interacted with:
-
+    
+    p A button is a type of control. They are rectangular, clickable interactive images within a scene. #[a(href='#the-gameclient') Game Clients] receive events when participants interact with a button.
+    
     ul
-        li `mousedown` - Emitted when a button is pressed by a participant.
-        li `mouseup` - Emitted when a button is released by a participant.
-
-    p Interactive developers can use buttons to allow participants to contribute to votes, cause actions to happen or even control in-game entities directly.
+        li `Down` event is sent when a button is pressed by a participant
+        li `Up` event is sent when a button is released by a participant
+    
+    .figure
+        img(src='/img/reference/interactive/button.png' alt='Sample button control which costs the participant one spark if they click on it.')
+    p.caption Sample button control which costs the participant one spark if they click on it.
+    
+    p You can use buttons to enable participants to vote, cause in-game actions to happen, or control in-game entities.
 
-    p Buttons have a number of properties which can be edited in both the Interactive Studio and from a Game Client.
+    p Buttons are customizable. These button properties can be edited from both the Interactive Studio and a Game Client:
     ol(style='list-style-type: decimal')
-        li Displayed text
-        li Spark Cost
-        li Progress Bar Width - Displayed at the bottom of a button.
-        li Disabled State - Disabled buttons cannot be interacted with.
-        li Cooldown Duration - Prevents interaction until it expires.
+        li Display text
+        li Spark cost (For more inof about sparks, go to #[a(href='#what-are-sparks') What are Sparks?]
+        li Width of the progress bar, which is displayed at the bottom of a button
+        li State of the button - Enable or disable them, disabled buttons cannot be interacted with
```

Comment:
"State of the button" would be used to refer to anything in this list.

A button has state, it can be updated:
- text
- disabled 
- etc

The disabled value is just one item of button state.